jnigen/example/in_app_java/lib/main.dart

```diff
@@ -11,8 +11,9 @@ import 'android_utils.dart';
 
 JniObject activity = JniObject.fromRef(Jni.getCurrentActivity());
 
-void showToast(String text) {
-  AndroidUtils.showToast(activity, text.jniString(), 0);
+/// Display device model number as Toast
+void showToast() {
+  AndroidUtils.showToast(activity, Build.MODEL, 0);
```

Comment:
I love it, just add a single thing to jnigen.yaml and boom new functionality in Dart!